.github/contributing.md

```diff
@@ -12,12 +12,11 @@ Make sure your pull request follows the guidelines/checklists in this document.
 - Double-check spelling and grammar.
 - No trailing whitespace is added to the end of any file.
 - One suggestion per pull request.
-- I understand that entries can be removed if they do not follow the guidelines.
+- Entries that repeatedly violate guidelines will be removed!
+- You have to [sign your commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
 
 ## Plugins/Integrations
 
-<!-- Ignore unless you're contributing to Plugins/Integrations -->
```

Comment:
Kinda liked these comments 🤔

Reply:
Those don't show up for people though, since it's the contribution doc not the PR template.

Reply:
fair enough